blockstore/splitstore/splitstore.go

```diff
@@ -181,6 +187,8 @@ func Open(path string, ds dstore.Datastore, hot, cold bstore.Blockstore, cfg *Co
 		cold:       cold,
 		markSetEnv: markSetEnv,
 
+		// nit: this doesn't need to be allocated as long as we don't move it.
+		// fyi, you _can_ return &s.txnViews.
```

Comment:
yeah, it doesn't move. I hadn't realized you can reuse the waitgroup with a new `Wait` after a previous `Wait` is done.

So yeah, no need to allocate indeed.